deebot_client/hardware/deebot/4vhygi.py

```
@@ -0,0 +1 @@
+p1jij8.py
```

Ruff

No such file or directory (os error 2)

(E902)